R/rdataretriever.R

```diff
@@ -103,7 +103,8 @@ install = function(dataset, connection, db_file=NULL, conn_file=NULL,
 #' names(vegdata$plant_comp_ok)
 #' }
 fetch = function(dataset, quiet=TRUE, data_names=NULL){
-    temp_path = tempdir()
+    temp_path = tolower(tempdir())
```

Comment:
If we use `temp_path = tolower(tempdir())`, this looks like we create a directory say `./myTempDir_xyz` and we then use the lower case `. ./mytempdir_xyz`. Let me know if I have it wrong

Reply:
Yes, it is doing exactly as you said. Please see #132.